### PR TITLE
Develop to next post 2.2.1

### PR DESCRIPTION
**Pull request by HanlinMiao**

### CODE REVIEW

**Comment by glennmatthews** [comment]
Should be able to remove `changes/5543.*` as that changeset was backported from `next` to `develop`.

File: nautobot/core/tests/test_views.py [hunk 1]
```diff
@@ -71,6 +73,26 @@ def test_search(self):
...
+
+        # Remove those models that are not searchable
+        existing_models = [model for model in existing_models if model not in GLOBAL_SEARCH_EXCLUDE_LIST]
+        existing_models.sort()
+
+        # See if there are any models that are missing from global search
+        difference = [model for model in existing_models if model not in global_searchable_models]
+        if difference:
+            self.fail(
+                f'Existing model/models {",".join(difference)} are not included in the searchable_models attribute of the app config.\nIf you do not want the models to be searchable, please include them in the GLOBAL_SEARCH_EXCLUDE_LIST constant in nautobot.core.constants.'
```
Comment (glennmatthews) [inline nautobot/core/tests/test_views.py:93]: We really should wrap this to 120 characters. Maybe better to do in `develop` than here though?

File: nautobot/docs/release-notes/version-2.2.md [hunk 2]
```diff
@@ -66,6 +66,53 @@ The CSV import functionality for all models has been changed from a synchronous
 Model CharFields' `max_length` attributes have been standardized globally to have at least 255 characters except where a shorter `max_length` is explicitly justified.
 
 <!-- towncrier release notes start -->
+## v2.2.1 (2024-04-15)
```
Comment (glennmatthews) [inline nautobot/docs/release-notes/version-2.2.md:69]: As a housekeeping item, we should also make sure that `develop` includes the v1.6.18 release notes.

**Review by glennmatthews — APPROVED** [review]

File: nautobot/core/tests/test_views.py (hunk 1 above)
Comment (HanlinMiao) [reply nautobot/core/tests/test_views.py:93]: I opened a PR for this comment and the one below https://github.com/nautobot/nautobot/pull/5584, should we merge that before this is merged or after?

Comment (glennmatthews) [reply nautobot/core/tests/test_views.py:93]: That can live in develop for now and bubble up to next after 2.2.2, IMHO.

**Pull request merged**